### PR TITLE
GUACAMOLE-1290: Add parameter for SSH public key.

### PR DESCRIPTION
**Pull request by necouchman**

### CODE REVIEW

File: guacamole-ext/src/main/resources/org/apache/guacamole/protocols/ssh.json [hunk 1]
```diff
@@ -38,6 +38,10 @@
                 {
                     "name"  : "passphrase",
                     "type"  : "PASSWORD"
+                },
+                {
+                    "name"  : "public-key"
```
Comment (mike-jumper) [inline guacamole-ext/src/main/resources/org/apache/guacamole/protocols/ssh.json:43]: Is there a corresponding translation for the field header?

**Review by mike-jumper — CHANGES_REQUESTED** [review]

File: guacamole-ext/src/main/resources/org/apache/guacamole/protocols/ssh.json (hunk 1 above)
Comment (necouchman) [reply guacamole-ext/src/main/resources/org/apache/guacamole/protocols/ssh.json:43]: Yeah, should probably include that. Fixed via rebase.

File: guacamole/src/main/frontend/src/translations/en.json [hunk 2]
```diff
@@ -741,6 +742,7 @@
         "FIELD_HEADER_PASSPHRASE"    : "Passphrase:",
         "FIELD_HEADER_PORT"          : "Port:",
         "FIELD_HEADER_PRIVATE_KEY"   : "Private key:",
+	"FIELD_HEADER_PUBLIC_KEY"    : "Public key:",
```
Comment (mike-jumper) [inline guacamole/src/main/frontend/src/translations/en.json:745]: Eek! A tab!

Otherwise, LGTM.

**Review by mike-jumper — CHANGES_REQUESTED** [review]

File: guacamole/src/main/frontend/src/translations/en.json (hunk 2 above)
Comment (necouchman) [reply guacamole/src/main/frontend/src/translations/en.json]: Whoa, not sure how that snuck in there. Fixed via rebase.

**Review by mike-jumper — APPROVED** [review]

**Pull request merged**